cibuildwheel/__main__.py

```diff
@@ -256,6 +256,7 @@ def build(self, options: Options, tmp_path: Path) -> None:
         ...
 
 
+# pylint: disable-next=inconsistent-return-statements
 def get_platform_module(platform: PlatformName) -> PlatformModule:
```

Comment:
FYI, I think the need for RET 503 below might be a bug in Ruff: https://github.com/charliermarsh/ruff/issues/3304

Reply:
It's been fixed, the noqa can go away in the next Ruff release :)